telegram/precheckoutquery.py

```diff
@@ -33,6 +33,8 @@ class PreCheckoutQuery(TelegramObject):
         currency (str): Three-letter ISO 4217 currency code
         total_amount (int): Total price in the smallest units of the currency (integer)
         invoice_payload (str): Bot specified invoice payload
+
+    Keyword Args:
```

Comment:
These are not `Keyword Args` in the sense of how documentation is generated. Please remove.
Also, `bot` is missing from the docstring, please add it..